GherkinSyncTool.Synchronizers.AzureDevOps/Utils/TextFilesEditMethods.cs

```diff
@@ -1,17 +1,58 @@
-﻿using System.IO;
+﻿using System;
+using System.IO;
 using System.Linq;
+using System.Reflection;
 using System.Text.RegularExpressions;
+using GherkinSyncTool.Models.Configuration;
+using GherkinSyncTool.Models.Utils;
+using GherkinSyncTool.Synchronizers.AzureDevOps.Model;
+using Microsoft.TeamFoundation.WorkItemTracking.WebApi.Models;
+using NLog;
 
 namespace GherkinSyncTool.Synchronizers.AzureDevOps.Utils
 {
     internal static class TextFilesEditMethods
     {
-        internal static void InsertLineToTheFile(string path, Regex lineRegex, string text)
+        private static readonly GherkinSyncToolConfig GherkinSyncToolConfig = ConfigurationManager.GetConfiguration<GherkinSyncToolConfig>();
+        private static readonly Logger Log = LogManager.GetLogger(MethodBase.GetCurrentMethod()?.DeclaringType?.Name);
+        internal static void InsertLineToTheFileRegex(string path, Regex lineRegex, string text)
```

Comment:
Shouldn't we move the implementation to the class with the same name located in GherkinSyncTool.Models.Utils?

Reply:
Done